README.md

```diff
@@ -1267,90 +1267,132 @@ The `job-id-regex` should contain one capture group while matching against the w
 
 Allows to execute jobs using HTCondor which is a specialized workload management system for compute-intensive jobs created by the Center for High Throughput Computing in the Department of Computer Sciences at the University of Wisconsin-Madison (UW-Madison).
```

Comment:
I'm not asking for this to be changed here as I don't think the required structure is in place yet. However, imagine a hypothetical world I'm about to describe and let me know what your thoughts are if that world were real.

At the moment the doc for the SGE backend is a weird hodge podge of SGE and the config backend. I think it'd be nice to just have an explanation for using the config backend in general and how it is generally useful to enable common HPC systems, with pointers to default setups in the reference.conf for specific examples.

If that were to be true, would you agree that the specialized docs for HTCondor wouldn't be necessary, or no? If no, what leads you to say that?